hr_employee_age/models/hr_employee.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    age = fields.Integer(
+        string='Age',
+        readonly=True,
+        compute='_compute_age'
+    )
+
+    @api.multi
+    @api.depends('birthday')
+    def _compute_age(self):
+        for record in self:
+            record.age = 0
+            if record.birthday:
+                record.age = relativedelta(
```

Comment:
Possibly, use a variable age = 0, and then finally assign to record.age = age, otherwise it will call write twice